touchforms/formplayer/static/formplayer/script/webformsession.js

```diff
@@ -299,7 +299,7 @@ WebFormSession.prototype.newRepeat = function(repeat) {
 
 WebFormSession.prototype.deleteRepeat = function(repetition) {
     var juncture = getIx(repetition.parent);
-    var rep_ix = +(repetition.rel_ix().split(":").slice(-1)[0]);
+    var rep_ix = +(repetition.rel_ix().replace('_',':').split(":").slice(-1)[0]);
```

Comment:
Pretty sure - the change is just allowing us to use the Java String representation for the subgroup start (the '_' character) in addition to the ':' used before